src/components/Person/Person.jsx

```diff
@@ -1 +1,19 @@
-// export const Person = ({ person }) => ();
+/* eslint-disable max-len */
+/* eslint-disable react/jsx-one-expression-per-line */
+import React from 'react';
+
+export const Person = ({ person }) => {
+  const { name, age, sex, isMarried, partnerName } = person;
+  const partnerMale = (sex === 'm') ? 'wife' : 'husband';
+
+  return (
+    <section className="Person">
+      <h2 className="Person__name">My name is {name}</h2>
+      {person.age && <p className="Person__age">I am {age}</p>}
```

Comment:
Instead of 'person.age', you can just use 'age' since it has already been destructured at the start of the component.